spec/loaded_file_spec.rb

```diff
@@ -11,27 +11,24 @@
   end
 
   it "can persist data about holdings file" do
-    file = build(:loaded_file)
-    file.save
-
-    expect(described_class.first[:filename]).to eq(file.filename)
+    expect { build(:loaded_file).save }.to change(described_class, :count).by(1)
   end
 
   describe "#latest" do
     it "returns the most recently loaded file of a given type" do
-      newest = create(:loaded_file, produced: Date.today - 1, type: "hathifile")
-      create(:loaded_file, produced: Date.today - 2, type: "hathifile")
-      create(:loaded_file, produced: Date.today - 1, type: "holding")
+      newest = create(:loaded_file, produced: Date.today - 1, type: "test")
```

Comment:
I was testing this locally, and the tests were fighting with data in the dev database. While we could (and perhaps eventually should) set up separate dev & test databases via `docker-compose.yml`, it's probably also worthwhile to make sure the tests don't make unnecessary assumptions or generate unnecessary conflict.